.versions

```diff
@@ -1,3 +1,3 @@
-ethereum:web3@0.5.0
+ethereum:web3@0.7.0
```

Comment:
this file is generated by meteor, when you publish as it contains all package dependencies.
We should remove it and i wouldn't rely on it as its updated by meteor, when you publish.